File: transport-remix/2/twoDC.m
function C = twoDC(x,y,t,M,D,x0,y0)
    
    [X Y] = meshgrid(x,y);
    C = X;
    for i=1:length(x)
        for j=1:length(y)
            C(j,i) = M/(4*pi*D*t(1)).*...
                     (exp(-(X(j,i)-x0).^2/(4*D*t(1))-(Y(j,i)-y0).^2/(4*D*t(1))));
        end
    end

end

File: transport-remix/particle-tracking/particleTracking/R/twoDC.R
twoDC <- function(x,y,ti,Dx,Dy,M=1,x0=0,y0=0){
    
    p <- expand.grid(x=x,y=y)
    C <- M/(4*pi*sqrt(Dx*Dy)*ti)*(exp(-(p$x-x0)^2/(4*Dx*ti)-(p$y-y0)^2/(4*Dy*ti)))

	C <- matrix(C,ncol=length(x),byrow=T)
	C
}